oauth2client/transport/httplib2.py

```diff
@@ -135,30 +133,24 @@ def make_authorized_http(credentials, http, refresh_status_codes):
     return _AuthorizedHttp(http, credentials, refresh_status_codes)
 
 
-def request(http, uri, method='GET', body=None, headers=None,
-            redirections=httplib2.DEFAULT_MAX_REDIRECTS,
-            **kwargs):
+class _ResponseWrapper(object):
```

Comment:
It's specific to httplib2, urllib3 and requests will already be in the right format. ;)